<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Geometry</class>
 <widget class="QWidget" name="Geometry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="buttonGroup" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="4">
       <widget class="QPushButton" name="Polyline">
        <property name="text">
         <string>Polyline</string>
        </property>
        <property name="icon">
         <iconset resource="GUI_PSN.qrc">
          <normaloff>:/images/polyline.png</normaloff>:/images/polyline.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QComboBox" name="AddList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="Circle">
        <property name="text">
         <string>Circle</string>
        </property>
        <property name="icon">
         <iconset resource="GUI_PSN.qrc">
          <normaloff>:/images/circle.png</normaloff>:/images/circle.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QPushButton" name="CrGroup">
        <property name="text">
         <string>Create Group</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="Rectangle">
        <property name="text">
         <string>Rectangle</string>
        </property>
        <property name="icon">
         <iconset resource="GUI_PSN.qrc">
          <normaloff>:/images/rec.png</normaloff>:/images/rec.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="Line">
        <property name="text">
         <string>Line</string>
        </property>
        <property name="icon">
         <iconset resource="GUI_PSN.qrc">
          <normaloff>:/images/line.png</normaloff>:/images/line.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Names of Group</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QPushButton" name="AddToG">
        <property name="text">
         <string>Add to Group</string>
        </property>
       </widget>
      </item>
      <item row="0" column="10">
       <widget class="QPushButton" name="Generate">
        <property name="text">
         <string>Generate</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>images/light.png</normaloff>images/light.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="10">
       <widget class="QPushButton" name="Exit">
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="Arc">
        <property name="text">
         <string>Arc</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="Ellipse">
        <property name="text">
         <string>Ellipse</string>
        </property>
        <property name="icon">
         <iconset resource="GUI_PSN.qrc">
          <normaloff>:/images/elipse.png</normaloff>:/images/elipse.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="ByPoint">
        <property name="text">
         <string>Create by point</string>
        </property>
       </widget>
      </item>
      <item row="0" column="9">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="8">
       <widget class="QPushButton" name="DelGr">
        <property name="text">
         <string>Delete Group</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="Tree">
        <attribute name="title">
         <string>Tree</string>
        </attribute>
        <widget class="QTreeWidget" name="treeWidget">
         <property name="geometry">
          <rect>
           <x>5</x>
           <y>1</y>
           <width>121</width>
           <height>381</height>
          </rect>
         </property>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </widget>
       <widget class="QWidget" name="workArea">
        <attribute name="title">
         <string>Work Area</string>
        </attribute>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>-1</x>
           <y>-1</y>
           <width>121</width>
           <height>381</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="Add"/>
        </widget>
       </widget>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="graphic"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="GUI_PSN.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Exit</sender>
   <signal>clicked()</signal>
   <receiver>Geometry</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>686</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>745</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
